.github/config.yml

```diff
@@ -12,9 +12,24 @@ newIssueWelcomeComment: |
 newPRWelcomeComment: |
   💖 Thanks for opening this pull request! 💖
 
-  ![typing cat](https://user-images.githubusercontent.com/2289/36400158-2c7c589e-1584-11e8-81c7-bd34fd3c392b.gif)
+  We use [semantic commit messages](https://conventionalcommits.org/) to streamline the Electron release process. Before 
+  your pull request can be merged, you must **update your pull request title** 
+  to start with a semantic prefix, OR prefix at least one of your commit messages.
```

Comment:
From an automation perspective it would be much easier to just use PR titles vs commit messages.

Reply:
Why? Can you elaborate?

Reply:
@zeke I was wrong.  I was thinking the automatic process only queries PRs, but it actually queries all the commits, so yeah, commits or PR titles are both good.